featuretools/tests/primitive_tests/test_rolling_primitive_utils.py

```diff
@@ -20,16 +20,16 @@
 from featuretools.tests.primitive_tests.utils import get_number_from_offset
 
 
-def test_get_rolled_series_without_gap(rolling_series_pd):
+def test_get_rolled_series_without_gap(window_series_pd):
```

Comment:
Do you think it would be more appropriate for this entire file to be named `test_time_series_primitive_utils.py` now?

Reply:
I think the name can stay the same. We still only test `rolling` functionality in that file, to my knowledge. I renamed the fixture in `conftest.py` to reuse it for the Expanding suite (and maybe Exponential and Time Series down the line).

Reply:
On that, I think we probably should add a targeted unit test to test the new `_apply_gap_for_expanding_primitives` function you added. This is currently tested in more complicated tests, but I think simple targeted tests for that specific functionality would be better - both that it raises an error with invalid input and that we get back the expected series with the proper input.